specification/network/resource-manager/Microsoft.Network/stable/2019-09-01/virtualWan.json

```diff
@@ -2942,6 +3176,10 @@
         "provisioningState": {
           "$ref": "./network.json#/definitions/ProvisioningState",
           "description": "The provisioning state of the virtual WAN resource."
+        },
+        "type": {
```

Comment:
Adding properties to existing responses is a breaking change and requires a new API version:
https://microsoft.sharepoint.com/:w:/t/azureresourcemanagerteam/EWXsAQ1yx25KkyYCeeWGUgwBSBxEdUDEbHi6FZ__U8EOQw?e=xEcppo

Reply:
@ryansbenson This is a branch for new API version

Reply:
Still getting used to how NRP does reviews :)